pkgs/unified_analytics/lib/src/analytics.dart

```diff
@@ -90,6 +98,7 @@ abstract class Analytics {
                   ? FileSystemStyle.windows
                   : FileSystemStyle.posix,
             ),
+        gaClient: FakeGAClient(),
```

Comment:
This comment isn't associated w/ a change in this PR, but perhaps annotate the containing method here (`Analytics.test`) w/ `visibleForTesting` from package:meta?

Reply:
hmm I'm not sure what you mean here. Would I be annotating the `sendData` method within the `FakeGAClient` class?

```dart
class FakeGAClient implements GAClient {
  @override
  String get apiSecret => throw UnimplementedError();

  @override
  String get measurementId => throw UnimplementedError();

  @override
  String get postUrl => throw UnimplementedError();

  @override
  http.Client get _client => throw UnimplementedError();

  @override
  void close() {}

  @override
  @visibleForTesting
  Future<http.Response> sendData(Map<String, Object?> body) =>
      Future<http.Response>.value(http.Response('', 200));
}
```

Reply:
I believe that `Analytics.test()` is visible from outside the package? The test factory ctor for use from testing? You'd just annotate that like:

```
...
/// [MemoryFileSystem] to use for testing
@visibleForTesting
factory Analytics.test({
  ...

```

YMMV as you already have the comment.

Reply:
Ah gotcha, that would be good to add in there as well, thanks for the heads up!